pandas/tests/series/indexing/test_datetime.py

```diff
@@ -667,3 +667,14 @@ def test_setitem_tuple_with_datetimetz():
     result[(0, 1)] = np.nan
     expected.iloc[0] = np.nan
     tm.assert_series_equal(result, expected)
+
+
+def test_indexing_regression():
```

Comment:
Could you give this test a more specific name related to the issue you are testing?

Reply:
Sure I'll rename it as `test_indexing_timeseries_regression` as this is issue with conversion of date range to time series. Let me know what you think.